src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/jira/StandaloneFormula.kt

```diff
@@ -224,6 +224,12 @@ class StandaloneFormula private constructor(
             )
         )
 
+        // For Jira pre 8.9.0 if the instance has no access to its own HTTP the dashboard view may freeze after log in.
```

Comment:
We can update `DataCenterFormulaIT` to cover 8.9.0

Reply:
If we are fine with adding this as another scenario (additional test time) then sure I can add it. If you have an idea how to add it and not increase test time then I'm also willing to implement it.

I'd suggest to add test for Jira 8.0.0 (8.9.0 is actually the first version not being affected) as this was the version that was failing in our internal tests. Alternatively we could test 8.5 as this was the LTS and would cover for another version than the one that we already have in tests of iJPT (not sure if this is correct direction of thinking)

Reply:
> how to add it and not increase test time

The test code is concurrent already (if we need to preserve the old variation).

Reply:
Oh, an important fact: We are in `StandaloneFormula`, so the test would be `StandaloneFormulaIT` and it doens't cover 8.0.0: https://github.com/atlassian/aws-infrastructure/blob/master/src/test/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/jira/StandaloneFormulaIT.kt

It has JSM 3.9.6 test only, which seems to not cover the problematic case (not sure why)

Reply:
Good point!

Reply:
Ok, I will ad the JSW 8.0.0 Standalone test. Hopwfully it will reproduce the problem and will show how the problem is fixed then.

- [ ] Add 8.0.0 Standalone IT

Reply:
Now that I started to write the test I'm realising that it's not so simple from the abstraction point of view.

If it's part of `StandaloneFormulaIT` should this test use VU? We didn't do that so far in `DataCenterFormulaIT` nor in `StandaloneFormulaIT` and I don't know about simpler way to log in into Jira inside JPT code. Log in is the operation that fails on Jira 8.0.0.

I currently don't see how to do this correctly in the current shape of the code.

Reply:
Speaking of the shape of the code and the IT: We wouldn't need ITs in this repository if the lib would promise behavior (that something will be done) rather than promise outcome (that something will work). This fix would not be needed if the contract of shaping permissions between instances would be given outside of the lib, however because we promise things to work (Jira application precisely) we are forced to manage the firewall here. 

I think that the performance test should be the IT for the "should work" part and this could be achieved by opening every single behavior of this lib into the API. I'm aware that it could enforce us to break API more, however this is not really unavoidable if we'd follow some other practices like versioning inside class's name.

Reply:
Perhaps infra could use VUs to test whether the provisioned Jira is workable. We'd need to resolve circular dependency though.

Reply:
I'm still not sure how this test would be implemented. I'm planning to skip it.

The access is required based on our documentation, so I'd say adding it doesn't require us to prove it's needed with test.